p2p/simulations/p2ptest.go

```diff
@@ -19,26 +19,33 @@ import (
 type myProtocol struct {
 	p2p.Protocol
 	proto int
+	peers map[*p2p.Peer]bool
```

Comment:
nice to add some comments for this var.

Reply:
p2ptest.go is for test，just show how to work with p2p.Protocol. 
Extra functions can be determined later, config-support for example